### ISSUE TITLE
Add example of using border color to show zero cases

### ISSUE THREAD

**Issue by zkamvar** [reconhub/incidence#115]
@annecori has brought up the fact that the default incidence plots are ambiguous as to when the start and end dates are (e.g. it's difficult to tell if the end of the epicurve contains zero case).

We should include examples in the plotting vignette and the documentation showing that users can supply `border = "black"` that will show the zero days

**Comment by thibautjombart** [comment]
I agree, but in fact this applies, in general, to any zero incidence (before the first cases, after the last one), i.e. whatever the range of the dates in the incidence object is.

**Comment by zkamvar** [comment]
> I agree, but in fact this applies, in general, to any zero incidence (before the first cases, after the last one), i.e. whatever the range of the dates in the incidence object is.

Yup, that's the point

**Comment by thibautjombart** [comment]
Sorry, misread the post, missed the 'start'
